src/app/static/src/app/components/plots/Choropleth.vue

```diff
@@ -150,10 +150,11 @@
 
                         const values = regionIndicators[area_id];
                         let value = values && values[indicator] && values[indicator]!!.value;
+                        const stringVal =  (value || value === 0) ? value.toString() : "";
```

Comment:
Also snuck in fix for this bug I noticed - 0 values weren't being shown in tooltips because falsy so were being rendered as empty strings